### PR TITLE
Update to Typescript 3

### LINKED ISSUE #602: typescript 3 support
TS v3.0 was released tonight https://blogs.msdn.microsoft.com/typescript/2018/07/30/announcing-typescript-3-0/

This issue is to track the `awesome-typescript-loader` compatibility with it.

### PR DESCRIPTION
**Pull request by domoritz**
Fixes #602

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "webpack-log": "^1.2.0"
   },
   "peerDependencies": {
-    "typescript": "^2.7"
+    "typescript": "^3"
```
Comment (sarink) [inline package.json]: I think you might want `^2.7 | ^3` (or whatever the syntax is), since it would still be totally valid to use TS 2.7 with this lib

Comment (domoritz) [reply package.json]: Updated.

**Comment by domoritz** [comment]
Thank you @sarink!

**Comment by Frank1e0927** [comment]
it means awsome-typescript-loader can use typescript3.0 with master branch??

**Pull request merged**